gitpod-io/browser-extension
# Communicate with Gitpod instances

## Description

Make sure to set the correct storage items in order to make in-app nudges for installing the extension work. 

## Related Issue(s)
Falls under EXP-623

## How to test

Go to a Gitpod instance (preferably not gitpod.io) and check that `sessionStorage` (from the devtools console) has a `browser-extension-installed` key in it.
